application/controllers/timers.py

```diff
@@ -14,77 +14,21 @@
     'timers': fields.List(fields.Nested(timer_fields)),
 }
 
-timer_post_parser = reqparse.RequestParser()
-timer_post_parser.add_argument(
-    'work_interval',
-    type=str,
-    required=True,
-    location=['json'],
-    help='work_interval parameter is required'
-)
-timer_post_parser.add_argument(
-    'rest_interval',
-    type=str,
-    required=True,
-    location=['json'],
-    help='rest_interval parameter is required'
-)
-
 class TimersResource(Resource):
```

Comment:
FANTASTIC trim down of this controller for just what we need in terms of MVP! Nice work figuring out how all of this communicated to understand what to take out.